File: Selenium/Activities/Activity_1.java
package Activity_CRM;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.Test;

public class Activity_1 {
    @Test
    public void testAssertFunctions() {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        System.out.println("done");
        String ExpectedString = driver.getTitle();
        System.out.println(ExpectedString);
        String ActualString = "SuiteCRM";
        System.out.println("DONE2");
        Assert.assertEquals(ExpectedString, ActualString);
        driver.close();
    }


}

File: Selenium/Project/Activity_1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("http://alchemy.hguy.co/crm")
    print("done")
    expected_string = driver.title
    print(expected_string)
    actual_string = "SuiteCRM"
    print("DONE2")
    assert expected_string == actual_string
    driver.quit()

if __name__ == "__main__":
    pytest.main()
